external/tree.h

```diff
@@ -440,8 +440,8 @@ template <typename T> class tree
 	}
 
 	//////////////////////////////////////////////////////////////////////////
-	iterator tree_iterator() const { return iterator( *(this) ); }
-	iterator tree_iterator() { return iterator( *(this) ); }
+	iterator get_tree_iterator() const { return iterator( *(this) ); }
```

Comment:
Could you please explain what is the source of name conflict?

Reply:
Compiler didn't like the fact that there was a class with the same name.

Reply:
OK. I'll report it to the author (he still works in Intel though)